src/UI/TextField.elm

```diff
@@ -492,8 +492,11 @@ genericAttr label isPlaceholder hasError width =
 
         else
             1
+
+    -- This Font attributes are the same of Subtitle2
     , Font.size 14
```

Comment:
Here too, for better design consistency, in line 477 you can reuse the `UI.Internal.Text.attributes` function to get text attributes.